kolibri/content/api.py

```diff
@@ -8,6 +8,7 @@
 from django.db.models import Q
 from kolibri.content import models as KolibriContent
 from kolibri.content.utils import validate
+from .constants import kinds
```

Comment:
Let's rename this one to `content_kinds`, since we have "kinds" of other things, as well.